Sources/XcodesKit/XcodeInstaller.swift

```diff
@@ -62,7 +62,7 @@ public final class XcodeInstaller {
             case let .unavailableVersion(version):
                 return "Could not find version \(version.appleDescription)."
             case .noNonPrereleaseVersionAvailable:
```

Comment:
Can't agree with you more! Can we go the next step and rename this enum as well?